<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaterielWindow</class>
 <widget class="QMainWindow" name="MaterielWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonAdd">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>idBadge</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonUpdate">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>update</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>parPage</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxParPage">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditId">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>id</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>241</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDateControle">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGetByIdBadge">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>getByIdBadge</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditIdBadge">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGetAll">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>getAll</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>page</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>dateControle</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxPage">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGetById">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>getById</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtongetCount">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>getCount</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Materiel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRemove">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>remove</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>nomMateriel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNomMateriel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>type</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxType">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGetByDateControle">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>getByDateControle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGetCountByDateControle">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>160</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>getCountByDateControle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGetCountByNomMateriel">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>310</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>getCountByNomMateriel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGetByNomMateriel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>getByNomMateriel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGetByType">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>370</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>getByType</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGetCountByType">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>370</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>getCountByType</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGetListIdBadge">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>getListIdBadge</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGetCountListIdBadge">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>getCountListIdBadge</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGetListDateControle">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>getListDateControle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGetCountListDateControle">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>190</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>getCountListDateControle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGetListNomMateriel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>340</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>getListNomMateriel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGetCountListNomMateriel">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>340</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>getCountListNomMateriel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGetListType">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>400</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>getListType</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGetCountListType">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>400</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>getCountListType</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNewDateControle">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>newDateControle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGetByNewDateControle">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>getByNewDateControle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGetCountByNewDateControle">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>220</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>getCountByNewDateControle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGetListNewDateControle">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>getListNewDateControle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGetCountListNewDateControle">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>250</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>getCountListNewDateControle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGetByPeriode">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>280</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>getByPeriode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGetCountByPeriode">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>280</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>getCountByPeriode</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxAvant">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>70</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Avant</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
